Code/GraphMol/Depictor/Wrap/rdDepictor.cpp

```diff
@@ -169,4 +209,55 @@ BOOST_PYTHON_MODULE(rdDepictor) {
        python::arg("permuteDeg4Nodes") = true,
        python::arg("bondLength") = -1.0),
       docString.c_str());
+
+  docString =
+    "Generate a depiction for a molecule where a piece of the \n\
+  molecule is constrained to have the same coordinates as a reference. \n\n\
+  This is useful for, for example, generating depictions of SAR data \n\
+  sets so that the cores of the molecules are all oriented the same way. \n\
+  ARGUMENTS: \n\n\
+  mol -    the molecule to be aligned, this will come back \n\
+           with a single conformer. \n\
+  reference -    a molecule with the reference atoms to align to; \n\
+                 this should have a depiction. \n\
+  confId -       (optional) the id of the reference conformation to use \n\
+  referencePattern -  (optional) a query molecule to be used to \n\
+                      generate the atom mapping between the molecule \n\
+                      and the reference. \n\
+  acceptFailure - (optional) if True, standard depictions will be generated \n\
+                  for molecules that don't have a substructure match to the \n\
+                  reference; if False, throws a DepictException.\n";
+  python::def("GenerateDepictionMatching2DStructure" ,
+	      RDDepict::GenerateDepictionMatching2DStructure,
+	      (python::arg( "mol" ) , python::arg( "reference" ) ,
+	       python::arg( "confId" ) = -1 ,
+	       python::arg( "refPatt" ) = 0 ,
```

Comment:
I think this should be `python::object()` so that it's initialized to `None` and it's clear from the python side that an int isn't expected.